triage/2023-12-18.md

```diff
@@ -0,0 +1,254 @@
+# 2023-12-19 Triage Log
+
+A lot of noise in the results this week; there was an lull in the noise
```

Comment:
`an lull`?

Reply:
I think it means: in the past weeks the perf. noise level went down and so were our thresholds. This week we had _more_ noise so 5 PRs triggered alarms.

Reply:
I think @Kobzol was pointing out it should be "a lull" rather than "an lull" :)

Reply:
Yeah :) TBH, I was also a bit confused by the word lull :laughing: